handler/packet.go

```diff
@@ -116,9 +116,11 @@ func PacketData(token, triggredIDColumnName, pipelineTableName, jobTableName str
 		glog.Infoln("New record ID for PACKET Pipeline:", id)
 		if pipelinedata.ID != 0 {
 			for j := range pipelineJobsdata {
-				sqlStatement := fmt.Sprintf("INSERT INTO %s (pipelineid, id, status, stage, name, ref, created_at, started_at, finished_at) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)"+
-					"ON CONFLICT (id) DO UPDATE SET status = $3, stage = $4, name = $5, ref = $6, created_at = $7, started_at = $8, finished_at = $9 RETURNING id;", jobTableName)
+				var jobLogURL string
+				sqlStatement := fmt.Sprintf("INSERT INTO %s (pipelineid, id, status, stage, name, ref, created_at, started_at, finished_at, job_kibana_url) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)"+
+					"ON CONFLICT (id) DO UPDATE SET status = $3, stage = $4, name = $5, ref = $6, created_at = $7, started_at = $8, finished_at = $9, job_kibana_url = $10 RETURNING id;", jobTableName)
 				id := 0
+				jobLogURL = Kibanaloglink(packetPipelineData.Sha, packetPipelineData.ID, packetPipelineData.Status, pipelineJobsdata[j].StartedAt, pipelineJobsdata[j].FinishedAt)
```

Comment:
if packetPipelineData.ID != 0 && len(pipelineJobsdata) != 0 {
jobLogURL = Kibanaloglink(packetPipelineData.Sha, packetPipelineData.ID, packetPipelineData.Status, pipelineJobsdata[j].StartedAt, pipelineJobsdata[j].FinishedAt)
}

Reply:
@chandankumar4  already check is there to get jobs , here also required ???